gulpfile.babel.js

```diff
@@ -0,0 +1,15 @@
+import gulp from 'gulp';
+import loadPlugins from 'gulp-load-plugins';
+const plugins = loadPlugins({
+  lazy: false,
+});
+
+gulp.task('lint', () =>
```

Comment:
nit: I know I'm probably in the minority on this, but I'm actually a fan of using the command-line version of eslint rather than the gulp plugin. In the package.json file we would have a 'scripts' section like:
```json
  "scripts": {
    "lint": "eslint src/**",
  },
```

At some point we'd have 'start' and 'test' scripts there as well